setup.py

```diff
@@ -297,26 +297,41 @@ def report(*args):
 cmake = CMake()
 
 
+def get_submodule_folders():
+    git_modules_path = os.path.join(cwd, ".gitmodules")
+    default_modules_path = [os.path.join(third_party_path, name) for name in [
+                            "gloo", "cpuinfo", "tbb", "onnx",
+                            "foxi", "QNNPACK", "fbgemm"
+                            ]]
```

Comment:
What's the default modules path for?

Reply:
It's there as a fallback, if .gitmodules is not available for some reason.